docs/Contributing/Issues/issues.md

```diff
@@ -0,0 +1,67 @@
+## Using the issue tracker
+
+The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs),
+features requests and for submitting pull
+requests, but please respect the following restrictions:
+
+* Please **do not** use the issue tracker for personal support requests. [The discussions tab](https://github.com/RI-SE/ATOS/discussions) on the repo can be used for this instead. 
+
+* Please **do not** derail or troll issues. Keep the discussion on topic and
+  respect the opinions of others.
+
+
+<a name="bugs"></a>
+## Bug reports
+
+A bug is a _demonstrable problem_ that is caused by the code in the repository.
+Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports:
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been
+   reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
+   latest `main` or development branch in the repository.
+
+3. **Isolate the problem** &mdash;
+A good bug report shouldn't leave others needing to chase you up for more
+information. Please try to be as detailed as possible in your report. What is
+your environment? What steps will reproduce the issue? What OS are you using, which
+browser is used for the gui?What would you expect to be the outcome? All these
+details will help people to fix any potential bugs.
+
+The issue report template is present in the issues tab on gihub, but the main points are also shown below:
+
+## Issue Template
+
+**Describe the bug**<br />
+A clear and concise description of what the bug is.
+ 
+**To Reproduce**<br />
+Steps to reproduce the behavior:<br />
+1. Go to '...'<br />
+2. Click on '....'<br />
+3. Scroll down to '....'<br />
+4. See error<br />
+
+**Expected behavior**<br />
+A clear and concise description of what you expected to happen.
+ 
+**Screenshots**<br />
+If applicable, add screenshots to help explain your problem.
+
+**Branch or release information (please complete the following information):**<br />
+ - Branch/release name: [eg. main, dev or version 1.0.0]
+ 
+**Desktop (please complete the following information):**<br />
+ - OS: [e.g. ubuntu, arch]<br />
+ - Browser [e.g. chrome, safari]<br />
+ - Version [e.g. 22]<br />
+
+**Additional context**<br />
+ Add any other context about the problem here.
```

Comment:
Is there any way to link to the template? Copies of the same text in different files might diverge.

Reply:
Yes I thought about this too. I left a copy here for now, but I think the easiest way would be to just link to the issues tab. As I understand readthedocs it needs to have the files in the same folder structure.

Reply:
I added a link to the template file in the repo instead.